.github/workflows/test-cygwin.yml

```
@@ -0,0 +1,113 @@
+name: Test Cygwin
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-minor-version: [7, 8, 9]
+        architecture: ["x86", "x86_64"]
+
+    timeout-minutes: 40
+
+    name: Python 3.${{ matrix.python-minor-version }} ${{ matrix.architecture }}
+
+    steps:
+      - name: Fix line endings
+        run: |
+          git config --global core.autocrlf input
+
+      - name: Checkout Pillow
+        uses: actions/checkout@v3
+
+      - name: Install Cygwin
+        uses: cygwin/cygwin-install-action@v2
+        with:
+          platform: ${{ matrix.architecture }}
+          packages: >
+            ImageMagick jpeg python3${{ matrix.python-minor-version }}-cffi
+            python3${{ matrix.python-minor-version }}-devel
+            python3${{ matrix.python-minor-version }}-numpy
+            python3${{ matrix.python-minor-version }}-cython liblapack-devel gcc-g++
+            python3${{ matrix.python-minor-version }}-sip
+            python3${{ matrix.python-minor-version }}-tkinter ghostscript
+            libfreetype-devel libimagequant-devel libjpeg-devel liblcms2-devel
+            libopenjp2-devel libraqm-devel libtiff-devel libwebp-devel libxcb-devel
+            libxcb-xinerama0 netpbm perl qt5-devel-tools xorg-server-extra zlib-devel
+            subversion make
```

Put in alphabetical order?

```suggestion
            ghostscript gcc-g++ ImageMagick jpeg make
            python3${{ matrix.python-minor-version }}-cffi
            python3${{ matrix.python-minor-version }}-devel
            python3${{ matrix.python-minor-version }}-numpy
            python3${{ matrix.python-minor-version }}-cython
            python3${{ matrix.python-minor-version }}-sip
            python3${{ matrix.python-minor-version }}-tkinter
            libfreetype-devel libimagequant-devel libjpeg-devel liblapack-devel
            liblcms2-devel libopenjp2-devel libraqm-devel libtiff-devel
            libwebp-devel libxcb-devel libxcb-xinerama0 netpbm perl
            qt5-devel-tools xorg-server-extra zlib-devel subversion
```

---

(I see you committed a change for this - feel free to mark as resolved. The button does not show up for me.)

---

The button shows up for me!

---

AFAIK it shows up for team members and PR author, but for some reason it does not show for the author of the comment.